src/lib/grammar.js

```diff
@@ -129,12 +129,13 @@ function buildRegex(map, keyEx, valEx) {
         '(?<' + keyEx + '>',
             keys,
         ')',
-        // value is optional
         '(?:\\(',
             '(?<' + valEx + '>',
                 GRAMMAR.VALUES,
             ')',
-        '\\))?',
+        '\\))',
+        // value is optional only for helper
+        keyEx === 'helper' ? '?' : ''
```

Comment:
So ideally we wanted to have all share the same syntax. But based on many discussions, we made Helper classes not require the parenthesis. So you can use `Bd` and `BdT` alone. But this would only be for helper classes as atomic classes DO require a param no matter what since the pattern is always prop(value). Also atomic classes have very generic name like just `D` for display, making the value optional for this would make the tool find a lot of false positives. It sucks, I know. :(